Sources/Graphiti/Federation/FederationResolver.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+import GraphQL
+import NIO
+
+public struct FederationEntityResolverArguments: Codable {
+    public let representations: [Map]
+}
+
+public protocol FederationResolver {
+    associatedtype Context
+    static var encoder: JSONEncoder { get }
+    static var decoder: JSONDecoder { get }
+    var sdl: String { get }
```

Comment:
Yea I would really prefer to not require the SDL since that requires users to put federation specific code into their schema. Maybe have a way of verifying the the supplied SDL to the built schema to make sure all types/fields/keys are included?